osx/KBKit/KBKit/System/KBSharedFileList.m

```diff
@@ -22,7 +22,6 @@ + (NSArray *)itemsForType:(CFStringRef)type {
   UInt32 seed = 0U;
   CFArrayRef itemsRef = LSSharedFileListCopySnapshot(fileListRef, &seed);
   NSArray *items = CFBridgingRelease(itemsRef);
-  CFRelease(itemsRef);
```

Comment:
It was an over-release that the static analyzer caught.
Above this is CFBridgingRelease which transfers it to ARC.